templates/json-stringify-module.js

```diff
@@ -0,0 +1,36 @@
+function jsonStringify(value) {
```

Comment:
Lightly modify from the JSON polyfill from Mozilla Developer Network: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON#Polyfill